frontend/src/components/dashboard/Onboarding.module.scss

```diff
@@ -0,0 +1,57 @@
+@import "~@mozilla-protocol/core/protocol/css/includes/tokens/dist/index.scss";
+@import "~@mozilla-protocol/core/protocol/css/includes/lib";
+
+.wrapper {
+  padding: $spacing-lg;
+  border-radius: $border-radius-md;
+  border: 1px solid $color-light-gray-40;
+
+  h2 {
+    @include text-title-2xs;
+    padding: $spacing-sm 0;
+  }
+
+  .steps {
+    display: flex;
+    flex-direction: column;
+    gap: $spacing-lg;
+    list-style-type: none;
+    margin: 0;
+    padding: $spacing-lg 0;
+    counter-reset: step-counter;
+
+    @media screen and #{$mq-md} {
+      flex-direction: row;
+    }
+
+    .step {
+      flex: 1 0 0;
+      display: flex;
+      flex-direction: column;
+      align-items: center;
+      gap: $spacing-xl;
+      background-color: $color-white;
+      border-radius: $border-radius-md;
+      padding: $spacing-lg;
+      counter-increment: step-counter;
+
+      &::before {
+        content: counter(step-counter);
+        background-image: url("../../../../static/images/dashboard-onboarding/shield.svg");
```

Comment:
Absolute paths work when the referenced files are in `/frontend/public/`. In this case, I tried to avoid duplicating files between the current and the React-based site, and am thus importing them from the root of the repository. After we migrate to the React-based one, however, we can copy over the files from `/static/` into `/frontend/public/`, and then we can turn these into absolute paths.

Reply:
Gotcha. Could you add a TODO for this take somewhere? It's completely non-blocking, but we should keep paths absolute where we can.

Reply:
Yep, added to MPP-1575!